solution

Stepan-TS opened the pull request:
[DEMO LINK](https://Stepan-TS.github.io/layout_search-bar-airbnb/)
[TEST REPORT LINK](https://Stepan-TS.github.io/layout_search-bar-airbnb/report/html_report/)

OleksiiBlezniuk commented on src/style.css:
```diff
@@ -0,0 +1,89 @@
+
+* {
+  box-sizing: border-box;
+  font-family: "Avenir-Book", sans-serif;
```
set font styles to `html` tag

OleksiiBlezniuk commented on src/style.css:
```diff
@@ -0,0 +1,89 @@
+}
+
+.image {
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  margin-top: 25px;
+  margin-left: 26px;
```
Why margins? This image has position: absolute and can be aligned by `top`, `bottom`, `right`, `left` properties

OleksiiBlezniuk commented on src/style.css:
```diff
@@ -0,0 +1,89 @@
+.small--input {
+  height: 42px;
+  padding-left: 33px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
```
These styles are already set to `input` class, no need to duplicate

OleksiiBlezniuk commented on src/style.css:
```diff
@@ -0,0 +1,89 @@
+  line-height: 19px;
+}
+
+.small--input:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
```
It's already set to `input`, no need to duplicate

OleksiiBlezniuk commented on src/style.css:
```diff
@@ -0,0 +1,89 @@
+  border-radius: 4px;
+}
+
+.small--input:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: "Avenir-Heavy", sans-serif;
+}
```
It's already set to input, no need to duplicate

OleksiiBlezniuk commented on src/style.css:
```diff
@@ -0,0 +1,89 @@
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-family: "Avenir-Heavy", sans-serif;
```
Set fonts to `html` tag and for all elements that not inherited it - use `inherit` value

OleksiiBlezniuk reviewed (changes requested):
You forgot to remove default outline
<img width="1431" alt="Screenshot 2023-01-26 at 14 52 03" src="https://user-images.githubusercontent.com/84275496/214840089-72a3dd45-8700-4192-8163-d48e33c25995.png">

Stepan-TS commented:
1.removed default outline;
2.seted font styles to html tag;
3.changed margin-top and margin-left to top and left;
4.removed styles from .input,.small--input:hover and removed .small--input:focus;
5.used inherit value for all elements that not inherited it;
